context/Web3Context/Web3Context.tsx

```diff
@@ -128,9 +133,38 @@ const Web3Store: React.FC<Web3ContextProps> = ({
         if (!isReady) {
             setEthBalance(0);
         }
+        else if(isReady){
+            triggerTutorial();
+        }
         return !!isReady;
     };
 
+    const triggerTutorial = async () => {
+        // If cookie with flag does not exist,
+        // start tutorial
+        const cookies = new Cookies();
+        if(cookies.get('tutorialCompleted') != 'true'){
+            addToast(['Trading with Tracer', `Click here to learn how to trade with Tracer`], {
+                appearance: 'info',
+                autoDismiss: false,
+            });
+            // Get the toast notification element
+            const toast = document.querySelector('.react-toast-notifications__container div[class^="Notification__Content-"]') as HTMLDivElement;
+            toast.addEventListener('click', function () {
+                const closeButton = document.querySelector('[class*="Notification__Close"]') as HTMLButtonElement;
+                closeButton.click();
+                setTourOpen(true);
+            });
+        }
+    };
```

Comment:
I know its called onboard.js but can we actually move this into the index.js page. This is because Web3Context is used in _app so its initiated on every page. Where as this is the tutorial for the trading page